### PR TITLE
'pkg' without args works like 'pkg .'

### LINKED ISSUE #73: `pkg` without arguments to work like `pkg .`

### PR DESCRIPTION
**Pull request by martinstarman**
close #73

### CODE REVIEW

**Comment by igorklopov** [comment]
(see 'Path to directory' below) instead of (see _Config_ section for more info)

**Comment by martinstarman** [comment]
PR updated.

File: readme.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ npm install -g pkg
 
 After installing it, run `pkg --help` without arguments to see list of options.
 
-The entrypoint of your project is a mandatory CLI argument. It may be:
+The entrypoint of your project is a optional CLI argument. It may be:
```
Comment (cvan) [inline readme.md]: `a optional` -> `an optional`

Comment (martinstarman) [inline readme.md]: Fixed. Thanks!

**Comment by ditta95aR** [comment]
#78

File: lib/index.js [hunk 2]
```diff
@@ -141,17 +141,13 @@ export async function exec (argv2) { // eslint-disable-line complexity
...
-      [ 'Pass --help to see usage information' ]);
-  }
   if (argv._.length > 1) {
     throw wasReported('Not more than one entry file/directory is expected');
   }
 
   // input
 
-  let input = path.resolve(argv._[0]);
+  let input = path.resolve(argv._.length ? argv._[0] : '.');
```
Comment (styfle) [inline lib/index.js]: Can this length check explicit? `argv._.length > 0 ? argv._[0] : '.'`

**Comment by igorklopov** [comment]
Hi! It is intentional, similar to ‘now’

**Pull request closed without merging**